test/unit/core/Object3D.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @author simonThiele / https://github.com/simonThiele
+ */
+
+module( "Object3D" );
+
+test( "getWorldScale", function() {
+  var obj = new THREE.Object3D();
+
+  checkIfDecomposeAndGetWorldScaleAreEqual(obj);
+
+  obj.scale.set(0, 0, 0);
```

Comment:
To test all ways of how an object can be scaled. It does not really matter in production code but should be checked what happens if a matrix is empty (other than the 1 on 3,3)